src-docs/src/views/accessibility/screen_reader.tsx

```diff
@@ -49,7 +49,9 @@ export default () => (
       <p>
         This link is visible to all on focus:{' '}
         <OuiScreenReaderOnly showOnFocus>
-          <OuiLink href="#">Skip navigation</OuiLink>
+          <OuiLink href="https://oui.opensearch.org/latest/">
+            Skip navigation
+          </OuiLink>
```

Comment:
<img width="600" alt="Screenshot 2023-04-18 at 09 42 21" src="https://user-images.githubusercontent.com/17855243/232666812-5f6b9c0c-36cb-4382-8ba9-760c452d1f80.png">

Reply:
This skip navigation link is for accessibility, and should not have the URL associated with it.

Reply:
Reverted changes

<img width="600" alt="Screenshot 2023-04-19 at 10 05 33" src="https://user-images.githubusercontent.com/17855243/232964399-c88b1020-6603-4cd2-82ab-d36fc32ce652.png">